hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -170,6 +171,33 @@ def lambda_handler(event, context):
     return {"statusCode": 200, "body": "Sure Yeah why not"}
 
 
+def sync_privacy_groups():
+    """
+    Implementation of the "sync_privacy_groups" function of threatexchange config.
+
+    1. call ThreatExchange API get_threat_privacy_groups_member
+    and get_threat_privacy_groups_owner to get the list of privacy groups
+
+    2. If the threat_updates_enabled is true, save it using config framework
+    """
+    lambda_init_once()
+    api_key = AWSSecrets.te_api_key()
+    api = ThreatExchangeAPI(api_key)
+    privacy_group_member_list = api.get_threat_privacy_groups_member()
+    privacy_group_owner_list = api.get_threat_privacy_groups_owner()
+    unique_privacy_groups = set(privacy_group_member_list + privacy_group_owner_list)
+
+    for privacy_group in unique_privacy_groups:
+        if privacy_group.threat_updates_enabled:
```

Comment:
maybe add a comment here like "HMA can only read from privacy groups that have threat_updates enabled. See here for more details: https://developers.facebook.com/docs/threat-exchange/reference/apis/threat-updates/v9.0"